docs/Developers/Useful_Links.rst

```diff
@@ -0,0 +1,47 @@
+Useful Links
+============
+
+The purpose of this document is to gather links that coala
+developers usually use throughout their work and therefore
+make it easier for everybody to find and access that specific
+information.
+
+.. note::
+
```

Comment:
I feel it`s important to be bold or in a box so that people would see it easier and notice that they can contribute. I feel that a separate paragraph is easier to be ignored in this case. ( you have just links, not a full doc ) I agree that using both is unnecessary